.asf.yaml

```diff
@@ -17,18 +17,6 @@
 
 notifications:
   pullrequests: commits@camel.apache.org
-  jira_options: link label comment
+  jira_options: link label=
 publish:
   whoami: asf-site
```

Comment:
This needs to be removed if we're not publishing from the `asf-site` branch

Reply:
Well, I agree it should be, but if there's no longer an asf-site branch in this repo it's just confusing rather than an error:-)

Reply:
This is read by the tools used to publish the website, see [chapter 5. Publishing](https://infra.apache.org/project-site.html), and might end up causing issues when publishing.